java-server-side-ui/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ version = "0.0.1-SNAPSHOT"
 java.sourceCompatibility = JavaVersion.VERSION_11
 
 extra["snakeyaml.version"] = "1.32"
```

Comment:
I would think we should remove the snakeyaml version override here, seems dated

Reply:
The snakeyaml overrides were added to fix a CVE. The CVE still persists because snakeyaml included the fix in minor release instead of a patch release. Since Spring Boot 2.7.x will only pick up patch releases, I assume we will have to keep the overrides as long as we are on 2.7.x

Reply:
I see now, can we bump the "snakeyaml.version" to 1.33?

Reply:
Yes, I'll do that for all the samples that use 1.32